languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt

```diff
@@ -136,9 +136,9 @@ class ServerRenderer @Inject constructor(
                         |   options: {
                         |      validate: {
                         |        <${if(it.hasBody()) "payload: ${it.bodies[0].type.toVrapType().simpleJoiName()}()," else ""}>
-                        |        params: {
+                        |        params: Joi.object({
```

Comment:
Pretty cool 👍  but i would also suggest to add it also to `params` 
https://github.com/commercetools/rmf-codegen/blob/master/languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt#L145 cause hapi will throw error if its javascript object, wdyt?